girder/test/test_cache.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2014 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import pytest
+
+from girder import _setupCache
+from girder.constants import SettingKey
+from girder.models.setting import Setting
+from girder.utility import config
+
+import mock
+
+
+@pytest.fixture
+def server(server):
+    # Extend server fixture to enable the cache
+    cfg = config.getConfig()
+    cfg['cache']['enabled'] = True
+    _setupCache()
+
+    yield server
+
+
+def testCachesAreAlwaysConfigured():
+    from girder.utility.cache import cache, requestCache
+
+    assert cache.is_configured
+    assert requestCache.is_configured
+
+
+def testSettingsCache(server):
```

Comment:
This test does not appear to use the request thread cache or other server components, but relies on the server fixture. Does the caching behavior itself necessarily rely on the server?

Reply:
No, it should be using the `db` fixture.